src/components/header/header-constants.tsx

```diff
@@ -0,0 +1,18 @@
+export const enum HeaderEnum {
+  default = 'default',
```

Comment:
I don't like this default idea.

I rather in the component we set it to false if nothing in the map matches.